<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reoshydraulicstructure2dTimeWindowWidget</class>
 <widget class="QWidget" name="Reoshydraulicstructure2dTimeWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="mExternallyCheckBox">
     <property name="toolTip">
      <string>Typically, used time window configured in original imported file.</string>
     </property>
     <property name="text">
      <string>Used Externally Defined Time Window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGroupBoxAuto">
     <property name="title">
      <string>Defined Automaticaly</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="ReosParameterDurationWidget" name="mStartOffset" native="true"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="mCombineCombo"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>The time window is defined by combining the time extents of the inflow bounday conditions. Duration offsets can be applyed to start and to end to adjust the final time window. </string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="ReosParameterDurationWidget" name="mEndOffset" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Combine method</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End offset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="mComboBoxOriginEnd"/>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="mComboBoxOriginStart"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ReosParameterDateTimeWidget" name="mStartTime" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ReosParameterDateTimeWidget" name="mEndTime" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReosParameterDurationWidget</class>
   <extends>QWidget</extends>
   <header>reosparameterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReosParameterDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>reosparameterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
